static/app/types/core.tsx

```diff
@@ -78,8 +81,11 @@ export enum DataCategory {
   REPLAYS = 'replays',
 }
 
-// https://github.com/getsentry/relay/blob/master/relay-common/src/constants.rs
-// Should be used in conjuction with DATA_CATEGORY_INFO rather than manipulating the string
+/**
+ * https://github.com/getsentry/relay/blob/master/relay-common/src/constants.rs
+ * Matches the backend singular backend enum directly.
+ * For display variations, refer to `DATA_CATEGORY_INFO` rather than manipulating these strings
```

Comment:
Within getsentry we have a handful of category related utility functions? Perhaps we should be moving those to sentry?

Reply:
That's a good point, I can start moving stuff over as I adopt this pattern of changes in getsentry. There will have to be a third PR to remove the deprecated type anyway.